renderer.js

```diff
@@ -44,6 +44,23 @@ function bindCanvas(c) {
   // renderer.gammaFactor = 2.2;
   renderer.shadowMap.enabled = true;
   renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+
+  // reload renderer WebGLInfo.reset function
+  // in order to continue chaning frames value
+  // in other case [.autoReset = false] it'll broke Avatars hair rendering [not sure why]
```

Comment:
This is worrying. Why would the stats affect the rendering? This should be debugged.